content/docs/09.5-security.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Security"
+metaTitle: "Spectro Cloud Security"
+metaDescription: "Get an overview of Palette's security awareness and controls and where yo can report security issues."
+icon: "lock"
+hideToC: false
+fullWidth: false
+---
+
+# Overview
+
+We view security as more than a feature. It is a fundamental aspect of our business and culture through transparency, continuous learning, and a security-first mindset. 
+
+By instilling a sense of collective responsibility for security, everyone at Spectro Cloud contributes to our overall security posture. Our dedication to security helps protect your interests and enhances the quality and reliability of our software and services.
+
+<br />
+
+## Security Culture and Awareness
+
+All employees at Spectro Cloud receive mandatory security training. Our developers receive additonal training focused on software security, as they are the first line of defense through secure coding practices. Our developers complete Open Worldwide Application Security Project (OWASP) Top 10 training to understand, identify and mitigate the most critical security risks and vulnerabilities that affect web applications.
```

Comment:
"Our developers" feels a bit repetitive (because of the previous sentence). Also, could you slightly rephrase so that you mention this OWASP training as one among others?

Reply:
@cloudmaniac Revised to say:

Our developers receive additional training focused on software security, as they are the first line of defense through secure coding practices, and they complete Open Worldwide Application Security Project (OWASP) Top 10 training to understand, identify and mitigate the most critical security risks and vulnerabilities that affect web applications.